src/SFA.DAS.ProviderApprenticeshipsService.Application/Services/ClientContentApiClient.cs

```diff
@@ -0,0 +1,33 @@
+﻿using SFA.DAS.ProviderApprenticeshipsService.Domain.Interfaces;
+using System.Threading.Tasks;
+using System.Net.Http;
+using SFA.DAS.Authentication.Extensions.Legacy;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Application.Services
+{
+    public class ClientContentApiClient : ApiClientBase, IClientContentApiClient
+    {
+        private readonly string ApiBaseUrl;
+
+        public ClientContentApiClient(HttpClient client, IContentClientApiConfiguration configuration) : base(client)
+        {
+            ApiBaseUrl = configuration.ApiBaseUrl.EndsWith("/")
+                ? configuration.ApiBaseUrl
+                : configuration.ApiBaseUrl + "/";
+        }
+
+        public async Task<string> Get(string type, string applicationId)
+        {
+            string banner = string.Empty;
```

Comment:
this variable can just be content. This service could be fetching any content based on the parameters passed to it. It just so happens that we only do this for the banner at the moment

Reply:
done